activity_calendar/models.py

```diff
@@ -421,8 +426,17 @@ def get_subscribed_users(self):
         return User.objects.filter(
             participant__activity_slot__parent_activity_id=self.parent_activity_id,
             participant__activity_slot__recurrence_id=self.recurrence_id,
+            participant__guest_name='',
         ).distinct()
 
+    def get_subscribed_guests(self):
```

Comment:
There's a bit of a mismatch between the return values of `get_subscribed_users` and `get_subscribed_guests`. The latter returns `Participants`, the former `Users`. Possibly refactor to have `get_subscribed_users` also return Participants?